tensorflow/lite/micro/examples/micro_speech/ceva/audio_provider.cc

```diff
@@ -60,39 +60,48 @@ TfLiteStatus GetAudioSamples(tflite::ErrorReporter* error_reporter,
 int32_t LatestAudioTimestamp() { return g_latest_audio_timestamp; }
 
 char filename[50] = "input.wav";
-FILE* infile;
-void init_audio() {
-  uint8_t mem[3];
-  printf("Using filename %s\n", filename);
-
-  infile = fopen(filename, "rb");
-  if (!infile) {
-    printf("Can't open file\n");
-    exit(1);
-  }
-
-  // skip wav header
-  for (int i = 0; i < 44; i++) {
-    fread(mem, 1, 1, infile);
-  }
+FILE *infile;
+void init_audio()
+{
+	uint8_t mem[3];
+	printf("Using filename %s\n", filename);
+
+	
+	infile = fopen(filename,"rb");
+	if (!infile)
+	{
+		printf("Can't open file\n");
+		exit(1);
+	}
+
+	//skip wav header
+	for (int i = 0; i < 44; i++)
+	{
+		fread(mem,1,1,infile);
+	}
 }
 
-void read_samples() {
-  int i = 0;
-  uint8_t mem[3];
-  bool done;
-  for (int i = 0; i < kNoOfSamples; i++) {
-    if (fread((char*)mem, 1, 2, infile) == 2) {
-      audio[i] = (int16_t)mem[0] + (((int16_t)mem[1]) << 8);
-    } else {
-      done = true;
-      fclose(infile);
-      infile = fopen(filename, "rb");
-      printf("EOF reached\n");
-
-      break;
-    }
-  }
+
+void read_samples()
+{
+	int i =0;
+	uint8_t mem[3];
+	bool done;
+	for (int i = 0; i < kNoOfSamples; i++)
+	{
```

Comment:
this seems to be counter to the formatting that we normally have.

Can you confirm that you're using -style=google with clang-format?

```
clang-format -i -style=google audio_provider.cc
```